src/go/k8s/apis/redpanda/v1alpha1/cluster_types_test.go

```diff
@@ -0,0 +1,48 @@
+package v1alpha1_test
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/vectorizedio/redpanda/src/go/k8s/apis/redpanda/v1alpha1"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func TestGetRedpandaResources(t *testing.T) {
+	resources := corev1.ResourceList{
+		corev1.ResourceCPU:    resource.MustParse("1"),
+		corev1.ResourceMemory: resource.MustParse("2Gi"),
+	}
+	rpkResources := corev1.ResourceList{
+		corev1.ResourceCPU:    resource.MustParse("0.5"),
```

Comment:
NIT: For me out would be easier if this value is not half of Resources. In the assertion I wasn't sure if calculation is right because of subtraction or simple assignment.